patches/frescobaldi-fix-invalid-metainfo-file.patch

```diff
@@ -0,0 +1,201 @@
++      <image type="source">https://user-images.githubusercontent.com/1644647/69188369-b280d980-0b1c-11ea-8f0e-e20b0f9f463d.png</image>
+     </screenshot>
+     <screenshot>
+       <caption>Syntax highlighting and automatic completion</caption>
+-      <image>https://user-images.githubusercontent.com/1644647/69151460-f7841c00-0ada-11ea-9577-1d2d358770ef.png</image>
++      <image type="source">https://user-images.githubusercontent.com/1644647/69151460-f7841c00-0ada-11ea-9577-1d2d358770ef.png</image>
+     </screenshot>
+     <screenshot>
+       <caption>Snippet manager and editor</caption>
+-      <image>https://user-images.githubusercontent.com/1644647/69151482-023eb100-0adb-11ea-9221-191052d5c92d.png</image>
++      <image type="source">https://user-images.githubusercontent.com/1644647/69151482-023eb100-0adb-11ea-9221-191052d5c92d.png</image>
+     </screenshot>
+     <screenshot>
+       <caption>Score Wizard to quickly setup a music score</caption>
+-      <image>https://user-images.githubusercontent.com/1644647/69151502-08cd2880-0adb-11ea-9c17-92886ef0a846.png</image>
++      <image type="source">https://user-images.githubusercontent.com/1644647/69151502-08cd2880-0adb-11ea-9c17-92886ef0a846.png</image>
+     </screenshot>
+   </screenshots>
+-
+-  <url type="homepage">http://frescobaldi.org/</url>
+-
+-  <launchable type="desktop-id">org.frescobaldi.Frescobaldi.desktop</launchable>
+-
+-  <translation type="gettext">frescobaldi</translation>
+-
+   <releases>
+-    <release version="3.1.2" date="2020-04-13">
+-      <description>
+-        <p>Bug fixes:</p>
+-        <ul>
+-          <li>Fixed the error "AttributeError: 'NoneType' object has no attribute 'cursor'"</li>
+-          <li>Fixed the "Two Pages (first page right)" mode</li>
+-          <li>Fixed pinch gesture zooming in Music View on Mac OS X</li>
+-          <li>Fixed printing the music score on Mac OS X</li>
+-          <li>Fixed cursor nagivation keyboard shortcuts on Mac OS X; new shortcuts:</li>
+-          <ul>
+-            <li>next document: ctrl+tab</li>
+-            <li>previous document: ctrl+shift+tab</li>
+-            <li>start of line: cmd+left</li>
+-            <li>end of line: cmd+right</li>
+-          </ul>
+-          <li>Fixed the display of filenames in document tabs on Mac OS X</li>
+-          <li>Fixed the global menu with no windows open on Mac OS X</li>
+-          <li>Fixed the error with convert-ly on in the Mac app bundle</li>
+-        </ul>
+-        <p>Improvements:</p>
+-        <ul>
+-          <li>German hyphenation patterns were added</li>
+-        </ul>
+-        <p>Translations:</p>
+-        <ul>
+-          <li>Dutch translation updated by Wilbert Berendsen</li>
+-          <li>Italian translation updated by Davide Liessi</li>
+-        </ul>
+-      </description>
+-    </release>
+-    <release version="3.1.1" date="2020-01-04">
++    <release type="stable" version="3.1.2" date="2020-04-13T00:00:00Z">
```

Comment:
While the validation doesn't complain, it's unusual to have a time. `date="2020-04-13"` ought to be enough

Reply:
It was added by appstreamcli utility, see [here](https://github.com/frescobaldi/frescobaldi/blob/master/Makefile#L6).

Reply:
ok.